### ISSUE TITLE
Hold January Style Council meeting and take follow up actions/publish

### ISSUE THREAD

**Issue by sarawilcox** [nhsuk/nhsuk-service-manual#794]
## What

Hold meeting, communicate outcomes and take follow up actions, including publishing any changes

## Dependent on

Preparing for the meeting: #792 

## Done when
- [x] Meeting held
- [x] Meeting notes shared on NHS.UK content channel
- [x]  Meeting notes written up and signed off by clinicians
- [x]  Amendments made to style guide and published
- [x] Amendments to PDFs page published - now in a separate ticket: https://github.com/nhsuk/nhsuk-service-manual/issues/963
- [x]  GitHub updated

**Comment by sarawilcox** [comment]
Minutes shared in NHS.UK Slack content channel.

**Comment by sarawilcox** [comment]
Final call in Slack for comments on write up of meeting and latest version of PDF guidance. I'll send to clinicians on Monday.

**Comment by sarawilcox** [comment]
Sent to clinicians for approval.

**Comment by sarawilcox** [comment]
Meeting notes approved. Waiting for feedback on PDF.

In the meantime, I've made the following changes:

- Added explanation and more examples to the section on hyphens and dashes
- Removed the readability tools page and explained that we do not recommend them on our health literacy page. Removed 2 broken links from health literacy page

These are covered in this PR: https://github.com/nhsuk/nhsuk-service-manual/pull/962
I've added the above to the What's new page, along with this note:
- Amended the site map page to bring it into line with the site map entry in the style guide.
